File: quick-start/node/create-agent.ts
import { config } from "dotenv";
import { PhonicClient } from "phonic";

config({ path: ".env.local" });

const client = new PhonicClient({
  apiKey: process.env.PHONIC_API_KEY,
  baseUrl: process.env.PHONIC_API_BASE_URL,
});

async function createAgent() {
  await client.agents.create({
    name: "my-first-agent",
    phone_number: "assign-automatically",
  });
}

createAgent();

File: 01-quick-start/python/create_agent.py
import os

from dotenv import load_dotenv
from phonic import Phonic

load_dotenv(".env.local")

client = Phonic(
    api_key=os.getenv("PHONIC_API_KEY"),
)


def create_agent():
    client.agents.create(
        name="my-first-agent",
        phone_number="assign-automatically",
    )


if __name__ == "__main__":
    create_agent()
